src/main/scala/io/github/kiemlicz/shelm/HelmPlugin.scala

```diff
@@ -30,6 +30,7 @@ object HelmPlugin extends AutoPlugin {
     lazy val repositories = settingKey[Seq[ChartRepository]]("Additional Repositories settings")
     lazy val shouldUpdateRepositories = settingKey[Boolean]("Perform `helm repo update` at the helm:prepare beginning")
     lazy val chartSettings = settingKey[Seq[ChartSettings]]("All per-Chart settings")
+    lazy val downloadedChartsCache = settingKey[Option[File]]("directory in which plugin will search for charts before downloading them")
```

Comment:
Please keep the style consistent with other settings - start with an uppercase letter

Reply:
Also: should there be a way to clear cache? It's going to grow endlessly